README.md

```diff
@@ -127,6 +127,10 @@ And just like that you're started!
 - [Upgrade Path](https://github.com/wwt/SwiftCurrent/blob/main/wiki/UPGRADE_PATH.md)
 - [Contributing to SwiftCurrent](https://github.com/wwt/SwiftCurrent/blob/main/.github/CONTRIBUTING.md)
 
+# Special Thanks
```

Comment:
My $0.02 is that we should move this below feedback. Also, why not just go ahead and call out the other dependencies?

Reply:
Consider making it clear these are test-only dependencies. It's a bonus for us that people recognize we do not have any production dependencies outside of Apple 1st party frameworks.

Reply:
I like the suggestion to include the others and specify as test-only. I had the section below feedback at first and I'm thinking I like that placement more as well.  - will update!